src/core/logger.lisp

```diff
@@ -4,15 +4,19 @@
   (:import-from #:alexandria
                 #:delete-from-plist)
   (:export #:*mito-logger-stream*
+           #:*mito-migration-logger-stream*
            #:with-sql-logging
            #:trace-sql
            #:*trace-sql-hooks*))
 (in-package :mito.logger)
 
 (defvar *mito-logger-stream* nil)
 
+(defvar *mito-migration-logger-stream* *standard-output*
```

Comment:
You should use a synonym-stream here. Currently you are binding the variable to the value of Standard output at the moment the code is loaded. See http://lisptips.com/post/127524780849/when-a-synonym-stream-is-useful

Reply:
Thanks !

rebased.